internal/utils/parser/token.go

```diff
@@ -67,13 +72,18 @@ func (t *tokenizer) ScanToken(data []byte, atEOF bool) (advance int, token []byt
 // token can be parsed as statement separator.
 //
 // Each statement is split as it is, without removing comments or white spaces.
-func Split(sql io.Reader) (stats []string) {
+func Split(sql io.Reader) (stats []string, err error) {
 	t := tokenizer{state: &ReadyState{}}
 	scanner := bufio.NewScanner(sql)
+
+	// Increase scanner capacity to support very long lines containing e.g. geodata
+	buf := make([]byte, maxScannerCapacity)
```

Comment:
```suggestion
	buf := make([]byte, 64 * 1024)
```

Shall we keep the default capacity at 64KB and let the buffer grow to new max capacity as needed? https://stackoverflow.com/a/37455465

In the future, we might also consider making max capacity configurable via env var if more users run into this issue.

Reply:
Perhaps even

```go
buf := make([]byte, 4096)
```

given the following setup from `bufio/scan.go`:

```go
const (
	// MaxScanTokenSize is the maximum size used to buffer a token
	// unless the user provides an explicit buffer with Scanner.Buffer.
	// The actual maximum token size may be smaller as the buffer
	// may need to include, for instance, a newline.
	MaxScanTokenSize = 64 * 1024

	startBufSize = 4096 // Size of initial allocation for buffer.
)
```

I'll push a fix :+1:

Reply:
Fix pushed via `--amend` on commit `fix: Increase buffer capacity (#470)`